x-pack/plugins/apm/typings/Span.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+interface DbContext {
```

Comment:
Here's where `DbContext` comes from! (re: my earlier review question) :) 

I think I am leaning toward either a) exporting things and then importing them explicitly where we use them, so we can trace where they come from ... orrrrr b) isn't there a way to do some kind of namespacing so things like "Stackframe" and "Context" become namespaced by Span, to help clarify and prevent problems with a type like "Context" eventually conflicting with something else?

Reply:
Good point with namespaces! I'm still super rusty in Typescript so will have to look that up, and whether we can disable the global namespace so we can force export/import.

Reply:
https://www.typescriptlang.org/docs/handbook/namespaces.html#namespacing